src/components/syncPlay/core/Manager.js

```diff
@@ -308,6 +308,7 @@ class Manager {
      * @param {Object} apiClient The ApiClient.
      */
     resumeGroupPlayback(apiClient) {
+        // TODO: rename this method, it's not clear what it does.
         this.followGroupPlayback(apiClient).then(() => {
```

Comment:
Should these methods be renamed in this PR?

Reply:
No, not really. I've left those there as a personal reminder. I'm not sure whether I was going to fix them in another open PR.
Is it ok to leave TODOs around like this? I can remove them otherwise.